loadtest.py

```diff
@@ -52,13 +66,19 @@ def get_payload_and_headers(size, compressed=False):
         payload = utils.compress(payload)
         headers['Content-Encoding'] = 'gzip'
 
+    # Make sure the final payload size is correct
+    assert len(payload) == size, (
+        'Payload is not the right size! %s vs. %s' % (len(payload), size)
+    )
```

Comment:
I moved this from `post_crash` which gets called every time we run a test to here which gets called every time we build a payload which happens at most 6 times. So... it's very slightly less processing, but more importantly it moves all the test correctness assertions into this function where it's easier to verify things.